src/stateManager/StateManager.ts

```diff
@@ -292,7 +264,8 @@ class StateManager {
 
             const validationResult =
                 await this.validationService.validateBlockConfirmation(
-                    blockConfirmation
+                    blockConfirmation,
+                    onChainTimestamp
```

Comment:
We need to meld this (`onChainTimestamp`) into the block model as soon as possible and try to use the block model as soon as possible - probably the first thing we should do is authenticate the block, so we just use our block model for the pipeline 

Currently if the channel is not opened, the block is queued without the onChainTimestamp, so the onChainTimestamp is lost